packages/@o3r/rules-engine/src/services/rules-engine.effect.ts

```diff
@@ -107,11 +111,13 @@ export class PlaceholderTemplateResponseEffect {
                 acc[paramName] = factset[paramName];
                 return acc;
               }, {});
-              replacements$.push(
-                this.translationService.translate(vars[varName].value, linkedParams).pipe(
-                  map((value: string) => ({ejsVar, value}))
-                )
-              );
+              if (this.translationService) {
```

Comment:
Shouldn't it be a bugfix on 8.0 ?

Reply:
Not really an issue as there is no application using the rules engine without the localization on the 8.0 and it will be integrated soon the in 8.1 :)